service_task_acc_test.go

```diff
@@ -79,7 +79,7 @@ var _ = Describe("Service Task", func() {
 
 			Eventually(func() bool {
 				t, errT = client.ServiceTask.Get(projectName, serviceName, t.Task.Id)
-				return t.Task.Success
+				return *t.Task.Success
```

Comment:
When the server is changed to actually work in async manner this will possibly be a dereference of a nil pointer.